server/routerlicious/packages/services-client/src/error.ts

```diff
@@ -86,7 +100,23 @@ export class NetworkError extends Error {
             message: this.message,
             canRetry: this.canRetry,
             isFatal: this.isFatal,
-            retryAfter: this.retryAfterMs,
+            retryAfter: this.retryAfterMs !== undefined ? this.retryAfterMs / 1000 : undefined,
```

Comment:
Is this.retryAfterMs !== undefined required? Will this work?
this.retryAfterMs ?  this.retryAfterMs / 1000 : undefined

Reply:
If `retryAfterMs === 0`, then `this.retryAfterMs ? this.retryAfterMs / 1000 : undefined` would show undefined when it should be 0